django-rgd/client/rgd_client/plugin.py

```diff
@@ -13,6 +14,49 @@ class RgdPlugin:
     def __init__(self, session: RgdClientSession):
         self.session = session
 
+    def download_checksum_file_to_path(
+        self, id: int, path: Optional[Path] = None, keep_existing=False, use_id=False
+    ):
```

Comment:
Sorry for the confusion, but this method should be placed in `CorePlugin`, not `RgdPlugin`. `RgdPlugin` is the base plugin, used mainly as a base class for other plugins, to aid in plugin discovery. `CorePlugin` is where the core RGD functionality resides.

Reply:
Ah, makes sense. Fixed this